.github/workflows/cluster-deploy-conditions-konto.yml

```diff
@@ -217,9 +245,17 @@ jobs:
           echo "condition=true" >> $GITHUB_OUTPUT
           echo ":soon: **prod-nest-forms-backend** " >> $GITHUB_STEP_SUMMARY
 
+      - id: prod-cvdmirror
+        name: Check for prod clamav
+        if: startsWith(github.ref, 'refs/tags/prod-cvdmirror') || steps.prod.outputs.condition == 'true'
+        run: |
+          echo "COUNT=$(($COUNT+1))" >> $GITHUB_ENV
+          echo "condition=true" >> $GITHUB_OUTPUT
+          echo ":soon: **prod-cvdmirror**" >> $GITHUB_STEP_SUMMARY
+
       - id: prod-clamav
         name: Check for prod clamav
-        if: startsWith(github.ref, 'refs/tags/prod-clamav') || steps.prod.outputs.condition == 'true'
+        if: startsWith(github.ref, 'refs/tags/prod-clamav') || steps.staging.outputs.condition == 'true'
```

Comment:
Why has `prod-clamav` staging condition?

Reply:
No it shouldn't be there. It's an old typo that I fixed in ClamAV branch afterwards, but this branch was created before the fix, so the typo remains. It`s now fixed.